target/linux/kirkwood/base-files/etc/board.d/01_leds

```diff
@@ -10,6 +10,9 @@ board_config_update
 board=$(board_name)
 
 case "$board" in
+globalscale,sheevaplug)
+	ucidef_set_led_timer "health" "health" "sheevaplug:blue:health" "200" "800"
```

Comment:
I suggest to use this led as status led (please see other kirkwood dts). It will be more useful.

Reply:
Done.